misc/vm_configs/scenarios/hybrid/nuc7i7dnb/misc_cfg.h

```diff
@@ -23,28 +23,23 @@
 				"loglevel=3 "	\
 				"i915.nuclear_pageflip=1 "	\
 				"hvlog=2M@0xe00000 "	\
-				"memmap=0x200000$0xe00000"
+				"memmap=0x200000$0xe00000 "	\
+				"maxcpus=3"
 
-#ifdef CONFIG_RDT_ENABLED
-/*
```

Comment:
could you please keep the comments inside CONFIG_RDT_ENBALED? This would help user to understand the meaning of MACRO definition.

Reply:
added comments in code